etc/taskcluster/decision_task.py

```diff
@@ -60,9 +57,10 @@ def tasks(task_for):
             "try-linux": [linux_tidy_unit, linux_docs_check, linux_release],
             "try-windows": [windows_unit, windows_arm64, windows_uwp_x64],
             "try-arm": [windows_arm64],
-            "try-wpt": [linux_wpt],
+            "try-wpt": [],
             "try-wpt-2020": [linux_wpt_layout_2020],
```

Comment:
`try-wpt-2020` also needs to be an empty list.

Reply:
Done.